dev/NavigationView/NavigationView.cpp

```diff
@@ -1533,6 +1514,17 @@ void NavigationView::UpdatePaneLayout()
                             const auto footersActualHeight = footerItemsRepeater.ActualHeight();
                             const auto menuItemsActualHeight = menuItems.ActualHeight();
 
+                            const auto paneFooterHeight = [this]() {
+                                if (const auto& paneFooter = m_leftNavFooterContentBorder.get())
+                                {
+                                    return paneFooter.ActualHeight();
```

Comment:
ActualHeight does not include Margin.Top and Margin.Bottom.  That's for instance a problem in the first MUXControlsTestApp NavigationView test page where paneFooter.Margin.Bottom is 4px (probably the new default).

You need to use 
const auto paneFooterMargin = paneFooter.Margin();

Then 
return paneFooter.ActualHeight() + paneFooterMargin.Top + paneFooterMargin.Bottom;